modules/nw-ovn-kubernetes-ansible-migration-about.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc
+
+[id="nw-ovn-kubernetes-ansible-migration-about_{context}"]
+= Using the Ansible plybook to migrate to the OVN-Kubernetes network plugin
+
+As a cluster administrator, you can use an Ansible playbook to migrate from the OpenShift SDN Container Network Interface (CNI) network plugin to the the OVN-Kubernetes plugin for your cluster.
+
+The Ansible playbook includes the following set of playbooks that each represent a stage in the migration process:
+
+. Prechecks
```

Comment:
Just to confirm these are playbooks and not plays? I see a difference mentioned in the Ansible docs.

![Uploading Screenshot From 2025-03-21 12-07-53.png…]()